### PR TITLE
hpc: Refactor hpc tests

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by Soulofdestiny** [comment]
verification: http://drnek.arch.suse.de/tests/792 to http://drnek.arch.suse.de/tests/797

File: lib/hpcbase.pm [hunk 1]
```diff
@@ -24,6 +24,12 @@ sub exec_and_insert_password {
     send_key "ret";
 }
 
+sub enable_and_start {
```
Comment (asmorodskyi) [inline lib/hpcbase.pm]: 1. enable_and_start_service ?
2. I think it worth to be added to utils.pm it is not HPC specific

Comment (okurz) [inline lib/hpcbase.pm]: it's fine like this for now. Can be extracted later - see #3263 as well

**Comment by okurz** [comment]
LGTM

**Pull request merged**